pkg/util/validation/notifications_limit_flag.go

```diff
@@ -6,57 +6,24 @@
 package validation
 
 import (
-	"encoding/json"
-	"fmt"
-
 	"github.com/pkg/errors"
-	"gopkg.in/yaml.v3"
 
 	"github.com/grafana/mimir/pkg/util"
 )
 
-var allowedIntegrationNames = []string{
-	"webhook", "email", "pagerduty", "opsgenie", "wechat", "slack", "victorops", "pushover", "sns", "webex", "telegram", "discord", "msteams",
-}
-
-type NotificationRateLimitMap map[string]float64
-
-// String implements flag.Value
-func (m NotificationRateLimitMap) String() string {
-	out, err := json.Marshal(map[string]float64(m))
-	if err != nil {
-		return fmt.Sprintf("failed to marshal: %v", err)
+var integrationsValidation = func(k string, _ float64) error {
```

Comment:
```suggestion
func validateIntegrationLimit(k string, _ float64) error {
```

Reply:
Done in https://github.com/grafana/mimir/pull/8378/commits/43f3a15f13e9d32b61e7d74638d746ef9b5ad3e6